apps/admin-be/package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "admin-be",
+  "version": "1.0.0",
+  "description": "backend for admin panel",
+  "main": "dist/index.js",
+  "license": "MIT",
```

Comment:
@teamoteh and I need to decide on a license, for now let's remove this field.

Reply:
Alright noted